src/components/SortControl/SortControl.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { render, fireEvent } from '@testing-library/react';
```

Comment:
It is not an issue, but usually we store tests like this:
It would be nice to update file structure this way.
```
components/
    ComponentName/
        ComponentName.tsx
        ComponentName.scss
        __tests__
             Component.test.tsx
```